torch/_meta_registrations.py

```diff
@@ -6083,6 +6083,15 @@ def meta_bucketize(self, boundaries, *, out_int32=False, right=False):
     ).contiguous()
 
 
+@register_meta([aten.histc])
+@out_wrapper()
+def meta_histc(input, bins=100, min=0, max=0):
```

Comment:
These ops have many more checks. Please see the behaviour in eager and add all the relevant checks.